src/sgame/sg_bot_util.cpp

```diff
@@ -557,6 +557,28 @@ AINodeStatus_t BotActionEvolve( gentity_t *self, AIGenericNode_t* )
 	return STATUS_FAILURE;
 }
 
+// Ideally this function would be part of equipment_t<upgrade_t>::unlocked(),
```

Comment:
Disagree with this comment since `unlocked` should just tell if the bot is legally allowed to have it. If a behavior tree requested buying it directly then `unlocked` should allow it even if there is no armor skill.

Reply:
I've removed the comment